src/components/LicenseFromPkg.test.tsx

```diff
@@ -15,9 +15,9 @@ describe("LicenseFromPkg", () => {
       name: "test-package",
       license: "MIT",
     };
-    expect(await render(<LicenseFromPkg pkg={pkg} />)).toContain(
-      "## 📜 License\n"
-    );
+    expect(
+      await render(<LicenseFromPkg pkg={pkg} title="License" />)
+    ).toContain("## License\n");
```

Comment:
Same here, this should be a separate test, not a change in the existing one.

Reply:
changes done .Pls check https://github.com/dbartholomae/jsx-readme/pull/56 @dbartholomae